android/app/src/main/java/com/zulipmobile/notifications/NotificationUiManager.kt

```diff
@@ -345,31 +345,10 @@ private fun updateNotification(
         setAutoCancel(true)
     }.build()
 
-    val summaryNotification = NotificationCompat.Builder(context, CHANNEL_ID).apply {
-        setGroup(groupKey)
-        setGroupSummary(true)
-
-        color = context.getColor(R.color.brandColor)
-        setSmallIcon(if (BuildConfig.DEBUG) R.mipmap.ic_launcher else R.drawable.zulip_notification)
-
-        // For the summary we use an "inbox-style" notification, as recommended here:
-        //   https://developer.android.com/training/notify-user/group#set_a_group_summary
-        setStyle(NotificationCompat.InboxStyle()
-            // TODO(#5115): Use the org's friendly name instead of its URL.
-            .setSummaryText(fcmMessage.identity.realmUri.toString())
-            // TODO: Use addLine and setBigContentTitle to add some summary info when collapsed?
-            //   (See example in the linked doc.)
-        )
-
-        // TODO Does this do something useful?  There isn't a way to open these summary notifs.
-        setAutoCancel(true)
-    }.build()
-
     NotificationManagerCompat.from(context).apply {
         // This posts the notifications.  If there is an existing notification
         // with the same tag and ID as one of these calls to `notify`, this will
         // replace it with the updated notification we've just constructed.
-        notify(groupKey, NOTIFICATION_ID, summaryNotification)
```

Comment:
Since we're no longer creating summary notifications, we should also remove the corresponding logic for them at the other end when notifications get removed because the messages were read. That can be found in `removeNotification`, above in this file.